server/container_stop.go

```diff
@@ -25,6 +25,9 @@ func (s *Server) StopContainer(ctx context.Context, req *pb.StopContainerRequest
 		if err := s.runtime.StopContainer(c, req.Timeout); err != nil {
 			return nil, fmt.Errorf("failed to stop container %s: %v", c.ID(), err)
 		}
+		if err := s.storageRuntimeServer.StopContainer(c.ID()); err != nil {
```

Comment:
We're now doing this in both `StopContainer` and `RemoveContainer`. The CRI API description seems to indicate that pod sandbox and container resources should be freed at `Stop` time. Do we want to keep removing them from both places ?